WG-CC/transitiontoolkit-frontend
# Will/UI login flow

# Description

I made two of the UI components based on the Figma. Right now the text is hardcoded. We need to look into how we want to import text around the app. A JSON makes the most sense, but we still need to see the best way to do it.

# Closes issue(s)

Resolves #8 

# How to test
Go to my branch and run the server. In the `App.tsx` file you can see the components. Check to see if they look like the Figma which is [here] (https://www.figma.com/design/Zh5emLgAjERi1jkl5vLuba/Transition-Toolkit?node-id=531-15697&t=z9qLIs45OtT8At9d-0 ).


# Changes include

1. Has the UI for the first two steps of the login flow

# Checklist
